pkg/blob/controllerserver.go

```diff
@@ -185,6 +190,9 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 	if !isSupportedContainerNamePrefix(containerNamePrefix) {
 		return nil, status.Errorf(codes.InvalidArgument, "containerNamePrefix(%s) can only contain lowercase letters, numbers, hyphens, and length should be less than 21", containerNamePrefix)
 	}
+	if isCacheEnabled {
+		// TODO: early failure here for dynamic provisioning, we should block any invalid account types
```

Comment:
Are current account types all supported? `Standard_LRS, Premium_LRS, Standard_GRS, Standard_RAGRS`

Reply:
This is TBD from the edge cache side, so this is a callout for that future work.